File: Source/Rio.Models/DataTransferObjects/Parcel/ParcelAllocationAndConsumptionDto.cs
using System.Collections.Generic;
using Rio.Models.DataTransferObjects.ParcelAllocation;

namespace Rio.Models.DataTransferObjects.Parcel
{
    public class ParcelAllocationAndConsumptionDto
    {
        public int ParcelID { get; set; }
        public string ParcelNumber { get; set; }
        public double ParcelAreaInAcres { get; set; }
        public List<ParcelAllocationDto> Allocations { get; set; }
        public List<ParcelMonthlyEvapotranspirationDto> MonthlyEvapotranspiration { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/parcel/parcel-allocation-and-consumption-dto.ts
import { ParcelMonthlyEvapotranspirationDto } from './parcel-monthly-evapotranspiration-dto.1';

export class ParcelAllocationAndConsumptionDto {
    ParcelID: number;
    ParcelNumber: string;
    ParcelAreaInAcres: number;
    WaterYear: number;
    AcreFeetAllocated: number;
    MonthlyEvapotranspiration: Array<ParcelMonthlyEvapotranspirationDto>;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}